client/js/config/constant.js

```diff
@@ -1 +1,5 @@
+export const CharityCategory = '../assets/category/Charity.png';
+export const EnviroCategory = '../assets/category/Enviro.png';
+export const PoliticoCategory = '../assets/category/Politico.png';
+export const SocialCategory = '../assets/category/Social.png';
 export const APOLLOCLIENTADDRESS = 'http://157.245.224.214:8000/';
```

Comment:
You can make it more readable renaming it to `APOLLO_AUTH_ADDRESS` for example.

Reply:
Done.